source/_static/lecture_specific/opt_tax_recur/recursive_allocation.py

```diff
@@ -1,289 +1,244 @@
-import numpy as np
-from scipy.interpolate import UnivariateSpline
-from scipy.optimize import fmin_slsqp
-from quantecon import MarkovChain
-from scipy.optimize import root
-
-
 class RecursiveAllocation:
```

Comment:
I don't understand the choice of class names in these lectures. In the lecture, allocations seem to be sequences. Are `RecursiveAllocation` and `SequentialAllocation` supposed to represent sequences?

Reply:
I am just keeping what was used originally but these names do not bother me too much. Quoting one sentence from the content: "After first presenting the model in a space of sequences, we shall represent it recursively in terms of two Bellman equations formulated along lines that we encountered in Dynamic Stackelberg models." It seems that the lecture is emphasizing the contrast between sequential representation and recursive representation. If you come up with better names for these representations, it would be great, but we would need to adjust the lecture content in a consistent way.

Reply:
Or do you mean you don't like "Allocation"s in the class names?

Reply:
How about `SequentialLS` and `RecursiveLS` standing for the sequential and recursive representations of the Lucas-Stokey model? The other lecture has `RecursiveAllocationAMSS` -- I can shorten that to `AMSS`.

Reply:
> Or do you mean you don't like "Allocation"s in the class names?

Yes -- I guess those should be classes that are meant to represent the model(s) described in the lecture.

Reply:
`SequentialLS` and `RecursiveLS` sound good if you insist. But I want to note that in the recursive representation, allocation is actually formulated in a recursive way (see the time invariant functions in eq. (32)), and I have to admit that it's not obvious to me yet how does changing these names improve the clearness.

Reply:
“The difference between the right word and the almost right word is really a large matter – it is the difference between the lightning and the lightning bug.” -- Mark Twain :)

Reply:
What I was saying is that I am not so sure why one is "right" and the other is "almost right", since "RecursiveAllocation" and "SequentialAllocation" form a good contrast in how the model solution is formulated as emphasized in the content, but it seems that you have a strong preference for what you are proposing, and I am indifferent between these. I will add this modification to this PR.